### DISCUSSION TITLE
Support for tailwind v4 and daisyUI v5

### DISCUSSION THREAD

**Discussion by kuchta** [themeselection/flyonui#9] [Ideas / Request new feature]
Are you considering supporting a new version of tailwind v4 and daisyUI v5 (alphas already available for both of them) and if so, when do you expect an alpha version to be released?

Thank you...

**Comment by PruthviPraj00** [comment]
Hello @kuchta,  

Thank you for your inquiry! We are considering support for the new versions of Tailwind v4 and DaisyUI v5. We appreciate your patience and enthusiasm, and we’ll keep you updated!  

Best regards

**Comment by PruthviPraj00** [comment]
Hello @kuchta,  
  
**FlyonUI 2.0.0** has been released, now with support for **Tailwind CSS v4**! 🚀
